### PR TITLE
[Chore] Setup project

### LINKED ISSUE #15: [Chore] Setup project
## Why

Setup repo
 
## Acceptance Criteria

Create git repo for Android "SMS Forwarder"

### PR DESCRIPTION
**Pull request by hoangnguyen92dn**
## What happened 🤔

Setup android project

## Insight 👀

- Rename application Id and package name
- Add PR template
- Init version `0.0.1`

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+
+## Insight 👀
+Describe in details how to achieve the target feature/fixes, referenced documentation is welcome as well.
+
+
+## PoW (a.k.a Proof Of Work)💪
+Show us the proof that it worked (photo, gif...etc)
+
+<img src="URL_GOES_HERE" width=200 />
```
Comment (lydiasama) [inline .github/pull_request_template.md:12]: Let's use the template from [here](https://github.com/nimblehq/git-template/blob/main/.github/PULL_REQUEST_TEMPLATE.md). This git template was updated.

**Review by lydiasama — APPROVED** [review]
Just one thing about the PR template.

File: .github/pull_request_template.md (hunk 1 above)
Comment (hoangnguyen92dn) [reply .github/pull_request_template.md:12]: @lydiasama We are going to use a new template for android. I copied the template from [Trung's PR](https://github.com/nimblehq/android-templates/blob/e0307e932ba03bd85d393e8cc2c9ad70eff154cb/.githubtemplate/PULL_REQUEST_TEMPLATE.md)

**Review by luongvo — APPROVED** [review]
lgtm

**Pull request merged**